Documentation changes for EC2 spot instances (#432)
giantswarm/docs: src/content/reference/gsctl/list-nodepools.md

The result will be a table of all node pools of a specific cluster with the following details in columns:

- `ID`:              Node pool identifier (unique within the cluster)
- `NAME`:            Name specified for the node pool, usually indicating the purpose
- `AZ`:              Availability zone letters used by the node pool, separated by comma
- `INSTANCE TYPE`:   EC2 instance type used for worker nodes

> - `ALIKE`:           Whether similar instance types are used within this node pool (eg if m5.xlarge is defined also m4.xlarge is possible)

Reviewer: ```suggestion
- `ALIKE`:           Whether similar instance types are used within this node pool (e. g. if `m5.xlarge` is defined also `m4.xlarge` is possible)
```

- `ON-DEMAND-BASE`:  Number of on-demand instances that this node pool needs to have until spot instances are used

> - `SPOT-PERCENTAGE`: Percentage of spot instances used once the on-demand base capacity is fullfilled. A number of 40 would mean that 60% will be on-demand and 40% will be spot instances.

Reviewer: ```suggestion
- `SPOT-PERCENTAGE`: Percentage of spot instances used once the on-demand base capacity is fulfilled. A number of 40 would mean that 60% will be on-demand and 40% will be spot instances.
```

- `NODES MIN/MAX`:   The minimum and maximum number of worker nodes in this pool
- `NODES DESIRED`:   Current desired number of nodes as determined by the autoscaler
- `NODES READY`:     Number of nodes that are in the Ready state in kubernetes

> - `SPOT INSTANCES`:  Number of spot instances in the node pool

Reviewer: ```suggestion
- `SPOT INSTANCES`:  Number of nodes using spot instances
```
